Calibration/HcalAlCaRecoProducers/plugins/ProducerAnalyzer.cc

```diff
@@ -1,20 +1,87 @@
 // system include files
+#include <memory>
+#include <string>
+#include <iostream>
 
 // user include files
 
-#include "Calibration/HcalAlCaRecoProducers/src/ProducerAnalyzer.h"
 #include "DataFormats/Provenance/interface/StableProvenance.h"
 #include "DataFormats/MuonReco/interface/Muon.h"
-#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
-#include "FWCore/MessageLogger/interface/MessageLogger.h"
-#include "Geometry/Records/interface/CaloGeometryRecord.h"
-#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
-#include "Geometry/HcalTowerAlgo/interface/HcalGeometry.h"
 #include "DataFormats/GeometryVector/interface/GlobalPoint.h"
 #include "DataFormats/HcalCalibObjects/interface/HOCalibVariables.h"
 #include "DataFormats/TrackReco/interface/TrackExtra.h"
 #include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+// collections
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
+#include "DataFormats/JetReco/interface/CaloJetCollection.h"
+#include "DataFormats/HcalCalibObjects/interface/HOCalibVariableCollection.h"
+#include "DataFormats/EgammaReco/interface/SuperCluster.h"
+#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/HcalTowerAlgo/interface/HcalGeometry.h"
+
 #include "RecoTracker/TrackProducer/interface/TrackProducerBase.h"
+#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
+
+namespace cms {
+
+  //
+  // class declaration
+  //
+
+  class ProducerAnalyzer : public edm::one::EDAnalyzer<> {
+  public:
+    explicit ProducerAnalyzer(const edm::ParameterSet&);
+    ~ProducerAnalyzer() override;
+
+    void analyze(const edm::Event&, const edm::EventSetup&) override;
+    void beginJob() override {}
+    void endJob() override {}
+
+  private:
+    // ----------member data ---------------------------
+    std::string nameProd_;
+    std::string jetCalo_;
+    std::string gammaClus_;
+    std::string ecalInput_;
+    std::string hbheInput_;
+    std::string hoInput_;
+    std::string hfInput_;
+    std::string Tracks_;
```

Comment:
According to the CMS code style rules, the members should start with a lowercase letter. This seems to be obeyed here except for the `Tracks_`